performancecopilot/pcp
# qa and check-vm Changes committed to git@github.com:kmcdonell/pcp.git 20190811

Ken McDonell (5):
      qa/admin/check-vm et al: changes for 32-bit Fedora 30
      scripts/pcp-push: small changes for *.sha files
      qa/1150: change guard as the test now works for Ubuntu 19.04
      qa/023: fix filter ... more fd's are possible
      qa/374: rework remote command logic

 qa/023                              |    2 
 qa/1150                             |   19 ++++----
 qa/374                              |   61 +++++++++++++++++++--------
 qa/admin/check-manifest             |    3 -
 qa/admin/other-packages/manifest    |   80 ++++++++++++++++++------------------
 qa/admin/other-packages/unavailable |    3 -
 scripts/pcp-push                    |    7 ++-
 7 files changed, 104 insertions(+), 71 deletions(-)

Details ...

commit 2972e09d6ae1c7744fda3f733406374345df3a5a
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Mon Aug 12 13:38:38 2019 +1000

    qa/374: rework remote command logic
    
    Need to be more careful when running commands as user pcpqa on the
    remote machine.  Especially if sudo does not work there, then we
    were failing this test silently with no clue as to why.

commit 3fda017fdf92ac67d96ca3ed30ae6d55b2cc7ead
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Mon Aug 12 07:11:49 2019 +1000

    qa/023: fix filter ... more fd's are possible
    
    The state of pmcd's fd pool is not deterministic, and I think recent
    changes around closing of help text files (for DSO PMDAs in particular)
    may have moved things a bit so add a couple of more acceptable values
    into the filter.

commit 10611d0c8908b80f8bbaf72f3675ae10e8aaff8c
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Mon Aug 12 07:10:02 2019 +1000

    qa/1150: change guard as the test now works for Ubuntu 19.04

commit 48034f4a1df72ed84fe4960bf090fad81915c7ff
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Aug 11 20:15:43 2019 +1000

    scripts/pcp-push: small changes for *.sha files

commit 6cc25a8d14ef062608651b7a8d50d76bb92a4b35
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Aug 11 20:09:18 2019 +1000

    qa/admin/check-vm et al: changes for 32-bit Fedora 30
    
    Also unquote 'perl(...)' specs ... only confuses the process when
    using check-vm -p.
